lib/src/lifecycle_module.dart

```diff
@@ -83,23 +107,45 @@ abstract class LifecycleModule {
         new StreamController<LifecycleModule>.broadcast();
     _didUnloadChildModuleController =
         new StreamController<LifecycleModule>.broadcast();
+    _willSuspendController = new StreamController<LifecycleModule>.broadcast();
+    _didSuspendController = new StreamController<LifecycleModule>.broadcast();
+    _willResumeController = new StreamController<LifecycleModule>.broadcast();
+    _didResumeController = new StreamController<LifecycleModule>.broadcast();
+
+    _logger = new Logger(name);
```

Comment:
Or just `_logger = new Logger('w_module');` and we include the name in each message:

```dart
_logger.warning('Module "$name" is already loaded, cannot load.');
```